realm/realm-library/src/androidTest/java/io/realm/internal/PivotTest.java

```diff
@@ -23,61 +23,62 @@
 
 public class PivotTest extends TestCase {
 
-    Table t = new Table();
+    Table t;
```

Comment:
Yes, when we move the loading from a static block to a method, it is not possible to allocate a `Table` statically.